data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1856 @@
+			choice
+				`	"Sure, let me take you to my ship."`
+				`	"I'm not headed there right now, sorry."`
+					decline
+			`	You give him the berth for your ship, and he tells you he'll meet you there soon before running off. A few minutes after you've reached the <ship>, you see him approaching, pushing a small crate, "filled to the brim with the posters" he tells you as you help him load it into your cargo. When you're done, you show him to his bunk, and prepare for the trip.`
+				accept
+			label resistance
+			`The spaceport of <origin> looks different than the usual Saryd ports: several large posters with some form of badge or logo are scattered about, some fallen to the ground, others still sticking to the walls. Picking one up to get a better look, you recognize the symbol to be that of the Lunarium Redoubt, two moons swallowing a dimming sun. "Up to any unlawful behavior, Captain <last>?" You turn to see an Arach standing there, looking at you. You recognize him as Tummug, one of the Lunarium leaders you met on Remote Blue. "Talk in your ship, could we?" You bring him to the <ship>, and seal the hatch. "Spreading propaganda material against the Heliarchy, I am. The purpose of those posters you found, that is," he explains. "Not the best recruitment strategy, but what I can do without risking too many of our members, it is. Available, are you? In need of transport to <planet>, to continue the work, I am."`
+			choice
+				`	"Of course, we'll leave right away."`
+				`	"Sorry, I'm not going that way right now."`
+					decline
+			`	"First, get the merchandise, I must," he says as he leaves your ship. He comes back a few minutes later, pushing a small crate, "filled to the brim with the posters" he tells you as you help him load it into your cargo. When you're done, you show him to his bunk, and prepare for the mission.`
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Tummug and the posters hasn't arrived in the system yet. Better depart and wait for it.`
+	on complete
+		payment 288000
+		conversation
+			`As you're still handling landing procedures, Tummug is organizing the posters in several smaller piles. "Make sure everyone has the same amount of posters, I must. Let any of them slack off, I will not," he banters. Your ship touches down on the hangar, and Tummug hands you <payment>. He makes a call, and minute after minute some Saryds start coming to your ship. You open the hatch, and he starts handing out a pile of posters to each one who comes in. The posters aren't all identical, though the imagery is pretty much the same in all of them: a dark, nine-pointed star, crumbling to pieces.`
```

Comment:
He introduces himself as Tummug in the opening of the convo if you haven't joined them yet.

``The spaceport of <origin> looks different than the usual Saryd ports: several large posters with some form of badge or logo are scattered about, some fallen to the ground, others still sticking to the walls. As you pick one up to get a better look, an Arach approaches you. "Captain <last> you are, yes?" You nod. "Tummug, I am called. Caught your eye, I see that our posters have. Part of a group promoting new cultural festivals and movies, I am. A tiresome task at times, repetitive even, but always worth it, in the end, helping spread word of new art pieces is."``

Admitteddly idk if that was the case when you wrote this comment, lots of little stuff committed since.